<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title></title>
  <meta name="Description" content="Comprehensive academic homepage of Dr. Vinod K. Gupta: microbiome, multi-omics, machine learning, publications, grants, teaching, and more.">
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="2575.7">
  <style type="text/css">
    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    p.p2 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; color: #0000e9; -webkit-text-stroke: #0000e9}
    p.p4 {margin: 0.0px 0.0px 12.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    p.p5 {margin: 0.0px 0.0px 12.0px 0.0px; font: 12.0px Times; color: #0000e9; -webkit-text-stroke: #0000e9}
    li.li1 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    li.li9 {margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000; min-height: 14.0px}
    span.s1 {font-kerning: none}
    span.s2 {text-decoration: underline ; font-kerning: none}
    span.s3 {font-kerning: none; color: #000000; -webkit-text-stroke: 0px #000000}
    span.s4 {text-decoration: underline ; font-kerning: none; color: #0000e9; -webkit-text-stroke: 0px #0000e9}
    span.s5 {text-decoration: underline ; font-kerning: none; -webkit-text-stroke: 0px #0000e9}
    span.s6 {-webkit-text-stroke: 0px #000000}
    ul.ul1 {list-style-type: disc}
  </style>
</head>
<body>
<p class="p1"><span class="s1">Vinod K. Gupta, Ph.D.</span></p>
<p class="p2"><span class="s2">About</span><span class="s3"> </span><span class="s2">Skills</span><span class="s3"> </span><span class="s2">Research Experience</span><span class="s3"> </span><span class="s2">Publications</span><span class="s3"> </span><span class="s2">Selected Impact</span><span class="s3"> </span><span class="s2">Patent &amp; Grants</span><span class="s3"> </span><span class="s2">Editorial &amp; Awards</span><span class="s3"> </span><span class="s2">Memberships</span><span class="s3"> </span><span class="s2">Conferences</span><span class="s3"> </span><span class="s2">Teaching</span><span class="s3"> </span><span class="s2">Education</span><span class="s3"> </span><span class="s2">References</span><span class="s3"> </span><span class="s2">CV</span><span class="s3"> </span><span class="s2">Contact</span></p>
<p class="p1"><span class="s1">VG</span></p>
<h1 style="margin: 0.0px 0.0px 16.1px 0.0px; font: 24.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Vinod K. Gupta, Ph.D.</b></span></h1>
<p class="p1"><span class="s1">Computational Systems Biologist · Microbiome · Data Science · Precision Medicine</span></p>
<p class="p4"><span class="s1">Rochester, Minnesota, USA<span class="Apple-converted-space">  </span>· <a href="mailto:vinodgupta299@gmail.com"><span class="s4">vinodgupta299@gmail.com</span></a></span></p>
<p class="p1"><span class="s1">MicrobiomeMulti-omicsMachine LearningMetaproteomicsHPC</span></p>
<p class="p5"><span class="s3"><b>Profiles:</b> <a href="https://www.linkedin.com/in/drvinodkgupta/"><span class="s5">LinkedIn</span></a> · <a href="https://scholar.google.co.in/citations?user=w4gWV6YAAAAJ&amp;hl=en"><span class="s5">Google Scholar</span></a> · <a href="https://orcid.org/0000-0003-1467-9993"><span class="s5">ORCiD</span></a> · <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/mybibliography/"><span class="s5">PubMed</span></a></span></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>About</b></span></h2>
<p class="p4"><span class="s1">With 15 years of experience in computational biology, I integrate metagenomics, metaproteomics, metabolomics, transcriptomics, and clinical data to uncover host–microbiome mechanisms in chronic diseases (rheumatoid arthritis, multiple sclerosis, colorectal cancer). I develop ML/AI models for treatment response prediction and disease stratification, and build reproducible bioinformatics tools such as GMHI, GMWI2, TaxiBGC, and BPGA. My focus is translational: converting complex datasets into actionable diagnostics, wellness monitoring, and targeted therapies.</span></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Skills</b></span></h2>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Domain</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Metagenomics &amp; Microbiome analysis; Amplicon &amp; WGS; Microbial genomics; Pan-genome analysis</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Multi-omics integration: transcriptomics, proteomics, metaproteomics, metabolomics</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Biomarker discovery; Precision medicine; Wellness monitoring</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Computational</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Machine Learning &amp; AI; Biostatistics; Methods &amp; tools development</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">R (expert), Python (advanced), Linux (advanced), Perl (basic)</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">HPC pipelines; Reproducible workflows; Cloud-ready data engineering</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Research Experience</b></span></h2>
<p class="p1"><span class="s1">Postdoctoral Research Associate — Mayo Clinic (10/2022–Present) Microbiomics Program, Center for Individualized Medicine</span></p>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Integrate microbiome (WGS, 16S), transcriptomics (RNA-Seq), proteomics (Mass Spec, Olink), metaproteomics, and metabolomics; design clinical studies; implement multi-omics analysis strategies.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Build ML models to predict treatment responses and disease phenotypes; develop indices and web tools (GMWI2, GMWI-webtool).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Co-supervise Ph.D., Masters, and undergraduate students; contribute to grant writing and preliminary data generation; provide analytical support to Proteomics Core.</span></li>
</ul>
<h4 style="margin: 0.0px 0.0px 16.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Projects Leading</b></span></h4>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">RA methotrexate response prediction (treatment-naïve patients).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">CRC stool metaproteomics: protein signature identification.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">MS: microbiome–host immune relationships after B-cell depletion.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">GMWI2 design and validation; kefir study in critically ill adults; prebiotic oligosaccharide evaluation; GMWI-webtool development.</span></li>
</ul>
<h4 style="margin: 0.0px 0.0px 16.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Projects Involved</b></span></h4>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Giant cell arteritis plasma proteomics; CSF proteomics in first-episode psychosis; RA immunological subtype analyses.</span></li>
</ul>
<p class="p1"><span class="s1">Postdoctoral Research Fellow — Mayo Clinic (10/2017–10/2022)</span></p>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Led GMHI development; deep-learning model for RA improvement; TaxiBGC pipeline for BGCs.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Contributed to plasma metabolomics in RA; CyTOF in PBC; nested patterns in healthy gut microbiomes.</span></li>
</ul>
<p class="p1"><span class="s1">Research Associate — CSIR–IICB (07/2016–03/2017)</span></p>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Human microbiome variation across geography/ethnicity/subsistence; PanGFR-HM web resource.</span></li>
</ul>
<p class="p1"><span class="s1">Research Fellow (Ph.D. student) — CSIR–IICB (04/2011–06/2016)</span></p>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Microbial genomes, pan-genome analysis, sequence statistics, phylogeny, KEGG/COG functional analysis.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Developed BPGA pipeline; supervised Masters students; conference presentations and manuscripts.</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Publications</b></span></h2>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>First/Co-first Author</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Gupta VK, Koller AM, Hur B, Davis JM III, Sung J. <i>Gut Microbiome Signatures Forecast Clinical Response to Methotrexate in Treatment-Naive Early Arthritis Patients</i>. (in prep.).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Gupta VK, Janda GS, Pump HK, Lele N, Cruz I, Cohen I, Ruff WE, Hafler DA, Sung J, Longbrake EE. <i>Alterations in Gut Microbiome-Host Relationships After Immune Perturbation in Patients With Multiple Sclerosis</i>. <b>NNI</b>, 2025.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Chang D#, <b>Gupta VK#</b>, Hur B, Cobo-López S, Cunningham KY, Han N, Lee I, Kronzer VL, Teigen LM, Karnatovskaia LV, Longbrake EE, Davis JM III, Nelson H, Sung J. <i>Gut Microbiome Wellness Index 2</i>. <b>Nat Commun</b>, 2024.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Gupta VK</b>, Rajendraprasad S, Ozkan M, Ramachandran D, Ahmad S, Bakken JS, Laudanski K, Gajic O, Bauer B, Zec S, Freeman DW, Khanna S, Shah A, Skalski JH, Sung J, Karnatovskaia LV. <i>Kefir in Critically Ill Adults</i>. <b>BMC Med</b>, 2024.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Gupta VK</b>, Bakshi U, Chang D, Lee AR, Davis JM III, Chandrasekaran S, Jin Y-S, Freeman MF, Sung J. <i>TaxiBGC</i>. <b>mSystems</b>, 2022.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Gupta VK</b>, Cunningham KY, Hur B, Bakshi U, Huang H, Warrington KJ, Taneja V, Myasoedova E, Davis JM III, Sung J. <i>Gut Microbial Determinants of CI-Improvement in RA</i>. <b>Genome Med</b>, 2021.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Gupta VK</b>, Kim M, Bakshi U, Cunningham KY, Davis JM III, Lazaridis KN, Nelson H, Chia N, Sung J. <i>GMHI</i>. <b>Nat Commun</b>, 2020.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Gupta VK</b>, Paul S, Dutta C. <i>Geography/Ethnicity/Subsistence Variations</i>. <b>Front Microbiol</b>, 2017.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Chaudhari NM#, <b>Gupta VK#</b>, Dutta C. <i>BPGA</i>. <b>Sci Rep</b>, 2016.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Gupta VK</b>, Chaudhari NM, Iskepalli S, Dutta C. <i>Prevotella Divergences</i>. <b>BMC Genomics</b>, 2015.</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Co-author</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Hur B, <b>Gupta VK</b>, et al. <i>ACPA-negative vs ACPA-positive RA multi-omics</i>. <b>Front Immunol</b> (under review).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Teigen LM, McCarter SJ, Ziegert Z, Staley C, Grant KM, <b>Gupta VK</b>, ... Ross OA. <i>Intestinal microbiota in Lewy body spectrum disease</i>. <b>Parkinsonism Relat Disord</b>, 2025.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Cunningham KY, Hur B, <b>Gupta VK</b>, et al. <i>Plasma Proteome Profiling in GCA</i>. <b>Ann Rheum Dis</b>, 2024.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Haroon H, Ho AMC, <b>Gupta VK</b>, et al. <i>CSF Proteomic Signatures in First-Episode Psychosis</i>. <b>J Psychiatr Res</b>, 2024.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Lee DH, Seong H, Chang D, <b>Gupta VK</b>, et al. <i>GMWI-based Prebiotic Indexing</i>. <b>npj Sci Food</b>, 2023.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Chang D, <b>Gupta VK</b>, Hur B, Cunningham KY, Sung J. <i>GMWI-webtool</i>. <b>Bioinformatics</b>, 2023.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Cobo-López S, <b>Gupta VK</b>, Sung J, Guimerá R, Sales-Pardo M. <i>Robust Nested Pattern in Healthy Microbiomes</i>. <b>PNAS Nexus</b>, 2022.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Hur B, <b>Gupta VK</b>, et al. <i>RA Plasma Metabolomics</i>. <b>Arthritis Res Ther</b>, 2021.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Jang JS, Juran BD, Cunningham KY, <b>Gupta VK</b>, et al. <i>CyTOF &amp; PBC</i>. <b>Sci Rep</b>, 2020.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Chaudhari NM, Gautam A, <b>Gupta VK</b>, et al. <i>PanGFR-HM</i>. <b>Front Microbiol</b>, 2018.</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Selected Publications with Impact</b></span></h2>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Nature Communications (2020) — GMHI</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Citations: 290+; &gt;5,000 metagenomes; 12 diseases + healthy cohorts.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Adopted in 15+ countries; GMWI-webtool translation.</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Nature Communications (2024) — GMWI2</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">20+ citations (first year); &gt;8,000 metagenomes across 25+ countries.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Enhanced accuracy in distinguishing healthy vs non-healthy states.</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Frontiers in Microbiology (2017)</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Citations: 1,100+; &gt;1,000 microbiomes; 20+ global populations.</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Scientific Reports (2016) — BPGA</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Citations: 950+; 27,000+ downloads; used in 100+ countries.</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Patent &amp; Grants</b></span></h2>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Patent</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Sung J, <b>Gupta VK</b>. <i>Assessing Gut Health Using Metagenome Data</i>. US Patent 20220367004 (2022).</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Grants &amp; Funding</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">LOI to Mayo Clinic Aging Center: <i>Stool microbiome to predict aging trajectories</i>.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">NIH R21 in preparation: <i>High-sensitivity, high-specificity metaproteome identification pipeline</i>.</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Editorial Roles &amp; Awards</b></span></h2>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Editorial (Ad Hoc Reviewer)</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">Arthritis Research &amp; Therapy; BMC Bioinformatics/Genomics/Microbiology; Clinical Microbiology Reviews; Frontiers in Microbiology; Journal of Translational Medicine; Microbiome; Scientific Reports; Signal Transduction and Targeted Therapy.</span></li>
</ul>
<h3 style="margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Awards &amp; Fellowships</b></span></h3>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">ASM Student &amp; Postdoctoral Travel Award (2022).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">ICTP Workshop Travel Award (2017).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">CSIR JRF-GATE 5-year fellowship (2011).</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">MHRD GATE 2-year fellowship (2009).</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Professional Memberships</b></span></h2>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">American Society for Microbiology</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Biomedical Engineering Society</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">International Society for Microbial Ecology</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Conferences &amp; Meetings</b></span></h2>
<p class="p1"><span class="s1">Recent &amp; Selected</span></p>
<ul class="ul1">
  <li class="li1"><span class="s6"></span><span class="s1">ACR Convergence 2025, Chicago — oral; ASM Microbe 2022 — oral; Understanding Microbiome 2022 — invited oral.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">Microbiome Meeting CSHL 2022 — poster; BMES 2021 — oral; IHMC 2021 — poster; MIT Microbiome 2021 — poster; Keystone 2021 — poster.</span></li>
  <li class="li1"><span class="s6"></span><span class="s1">ACR Convergence 2020 — poster; multiple 2018–2019 posters (CSHL, Mayo, IHMC, Keystone).</span></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Teaching &amp; Mentorship</b></span></h2>
<p class="p4"><span class="s1">Mentored undergraduate, Masters, and Ph.D. trainees; supported thesis design, data analysis, interpretation, and manuscripts. Taught bioinformatics tools (genome/protein sequence analysis) at Vidyasagar College; delivered invited lecture on “Fundamentals of Microbiome Data Analysis” (JIS Institute, Kolkata).</span></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Education</b></span></h2>
<ul class="ul1">
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>Ph.D. (Bioinformatics, Microbiome, Genomics)</b> — CSIR-IICB / AcSIR, India (2017). Thesis: <i>Divergences in Gene Repertoires of Microbiome Components Derived from Distinct Body-sites of Human</i>.</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>M.S. Pharm (Pharmacoinformatics)</b> — NIPER Kolkata, India (2009).</span></li>
  <li class="li1"><span class="s6"><b></b></span><span class="s1"><b>B. Pharm</b> — Mohan Lal Sukhadia University, Udaipur, India (2009).</span></li>
</ul>
<ul class="ul1">
  <li class="li9"><span class="s1"></span><br></li>
</ul>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>CV</b></span></h2>
<p class="p4"><span class="s1">Download the full CV (PDF):</span></p>
<p class="p5"><span class="s2"><a href="file:///Users/vinodgupta/Documents/github_personal/vinod_full_site/assets/VinodGupta_CV_Vinod.pdf">Download CV</a></span></p>
<p class="p4"><span class="s1">Last updated 2025-08-16 (per document).</span></p>
<h2 style="margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000"><span class="s1"><b>Contact</b></span></h2>
<p class="p5"><span class="s3"><b>Email:</b> <a href="mailto:vinodgupta299@gmail.com"><span class="s5">vinodgupta299@gmail.com</span></a></span></p>
<p class="p4"><span class="s1"><b>Location:</b> Rochester, Minnesota, USA</span></p>
<p class="p1"><span class="s1">© 2025 Vinod K. Gupta · Built with plain HTML/CSS · GitHub Pages ready</span></p>
</body>
</html>
